### PR TITLE
Copy the web console image into the crio storage layer

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by smarterclayton** [comment]
@runcom

File: sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml [hunk 1]
```diff
@@ -101,11 +101,20 @@ extensions:
           make build-images
           docker tag openshift/origin-metrics-server:latest "openshift/origin-metrics-server:$( cat /data/src/github.com/openshift/aos-cd-jobs/ORIGIN_COMMIT )"
         fi
+    - type: "script"
+      title: "build the origin web console server container image"
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: or bite the bullet and inherit from the built release + ecosystem

Comment (smarterclayton) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: don't have the time right now but i agree.

**Comment by runcom** [comment]
+1

**Pull request merged**